### PR TITLE
[15.0][FWP] product_analytic

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Forward port https://github.com/OCA/account-analytic/pull/570

### CODE REVIEW

File: product_analytic/models/account_move.py [hunk 1]
```diff
@@ -19,14 +19,13 @@ class AccountMoveLine(models.Model):
...
-                )
-                ana_account = ana_accounts[INV_TYPE_MAP[inv_type]]
-                line.analytic_account_id = ana_account.id
+        inv_type = self.move_id.move_type
+        if self.product_id and inv_type and inv_type in INV_TYPE_MAP.keys():
+            ana_accounts = (
+                self.product_id.product_tmpl_id._get_product_analytic_accounts()
+            )
+            ana_account = ana_accounts[INV_TYPE_MAP[inv_type]]
+            self.analytic_account_id = ana_account.id
```
Comment (MiquelRForgeFlow) [inline product_analytic/models/account_move.py]:
```suggestion
        for line in self:
            inv_type = line.move_id.move_type
            if line.product_id and inv_type and inv_type in INV_TYPE_MAP.keys():
                ana_accounts = (
                    line.product_id.product_tmpl_id._get_product_analytic_accounts()
                )
                ana_account = ana_accounts[INV_TYPE_MAP[inv_type]]
                line.analytic_account_id = ana_account.id
```

Comment (JasminSForgeFlow) [reply product_analytic/models/account_move.py]: Changes updated, 
Thanks

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM! 👍🏽

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**